pygmt_helper/pygmt_helper/examples/srf_plotting.py

```diff
@@ -0,0 +1,89 @@
+from pathlib import Path
+
+import pandas as pd
+import numpy as np
+
+from pygmt_helper import plotting
+from qcore import srf
+
+
+map_data_ffp = Path("/path/to/qcore/qcore/data")
+output_dir = Path("output/dir")
+
+srf_ffp = Path("/path/to/the/srf/file")
+
+# Load the srf data
+planes = srf.read_header(str(srf_ffp), idx=True)
+corners = srf.get_bounds(str(srf_ffp), depth=True)
+corners = np.transpose(np.array(corners), (1, 2, 0))
+slip_values = srf.srf2llv_py(str(srf_ffp), value="slip")
+
+# Set map_data to None for faster plotting without topography
+# map_data = plotting.NZMapData.load(map_data_ffp, high_res_topo=False)
+map_data = None
```

Comment:
Maybe create an option for the map_data variable here? Perhaps a CLI option --use-map-data? Is this a CLI script at all? If so, I would prefer if the path, map data and srf filepaths were read in from command line arguments.

Reply:
This isn't actually a script that you would run, just an example of how to utilise the pygmt helper for srf plotting to help get people started when using it with their other scripts / functions. If it was a script then definitely for sure!

Reply:
Yeah as @joelridden mentioned this is not a command line script. In general I want to avoid directly writing scripts, because this leads to the case of the PhD students & postdocs coming back with modifications they need for their specific use case. 
Instead I believe the best approach is to create tools (i.e. packages) for them to integrate into their own code, and to make this easier its nice to have an example "script" that gives an idea of how to use the functionality of the package. Hopefully that makes, happy to discuss tomorrow.